src/NosCore.Controllers/CommandPacketController.cs

```diff
@@ -1,18 +1,65 @@
-﻿using System.Linq;
+﻿using System.Collections.Generic;
+using System.Linq;
 using System.Reflection;
 using JetBrains.Annotations;
 using NosCore.Core.Extensions;
+using NosCore.Core.Networking;
 using NosCore.Core.Serializing;
+using NosCore.Data;
+using NosCore.Data.WebApi;
 using NosCore.GameObject;
 using NosCore.GameObject.ComponentEntities.Extensions;
+using NosCore.GameObject.Networking;
 using NosCore.Packets.CommandPackets;
+using NosCore.Packets.ServerPackets;
 using NosCore.Shared.Enumerations;
+using NosCore.Shared.Enumerations.Map;
+using NosCore.Shared.I18N;
 
 namespace NosCore.Controllers
 {
 	[UsedImplicitly]
     public class CommandPacketController : PacketController
 	{
+	    public void Shout(ShoutPacket shoutPacket)
+	    {
+	        if (shoutPacket?.Message == null)
```

Comment:
useless check it shouldnt be accepted by the packet factory. if it can fix it and add a test